tests/wpt/mozilla/tests/css/perspective_zero.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+    <head>
```

Comment:
Updated - although it was definitely running that test locally without that line. Just out of curiosity, why do we need to list it both there are also (twice) in the manifest file?